src/assets/icon/IconCheck.js

```diff
@@ -16,7 +16,7 @@ function IconCheck(props) {
           fillRule="evenodd"
           clipRule="evenodd"
           d="M17.774 7.713a.75.75 0 01.013 1.061l-7.325 7.5a.75.75 0 01-1.109-.04l-3.175-3.75a.75.75 0 111.144-.969l2.643 3.121 6.748-6.91a.75.75 0 011.061-.013z"
-          fill="#fff"
+          fill={props?.fillIcon || '#fff'}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of optional chaining (`props?.fillIcon`) is unnecessary here because `props` will always be defined in a functional component. You can directly access the property as `props.fillIcon`.
```diff
- fill={props?.fillIcon || '#fff'}
+ fill={props.fillIcon || '#fff'}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->